cordova-lib/cordova-lib.js

```diff
@@ -18,19 +18,35 @@
 */
 
 // For now expose plugman and cordova just as they were in the old repos
+
+
+function addProperty(obj, property, modulePath) {
```

Comment:
util.js has a side effect in global code of invoking a mkrdir. Also, the addModuleProperty is not quite the same and it's used in a number of places. I have already created a shared version in cordova-common.